File: DSA/Shubham_590016506/Day_9_Question_2.c
#include <stdio.h>
#include <ctype.h>

int main() {
    const char *text = "Programming";
    int vowels = 0, consonants = 0;

    for (int i = 0; text[i] != '\0'; i++) {
        char ch = tolower(text[i]);
        if (isalpha(ch)) {
            if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u')
                vowels++;
            else
                consonants++;
        }
    }

    printf("Vowels: %d, Consonants: %d\n", vowels, consonants);
    return 0;
}

File: DSA/akashyadav_590014126/Day_9_Question_2.py
def frequency_of_vowels_consonants(s):
    vowels = "aeiou"    #all the vowels
    vowel_count = 0 #to count vowels
    consonant_count = 0 #to count consonents

    for char in s:
        if char.isalpha():  # check if the character is a letter
            if char.lower() in vowels:
                vowel_count += 1    #counting vowels
            else:
                consonant_count += 1    #counting consonents

    print(f"Vowels: {vowel_count}, Consonants: {consonant_count}")

#Example
frequency_of_vowels_consonants("Programming")   # Vowels: 3, Consonants: 8